webapp/src/pages/boardPage.tsx

```diff
@@ -143,7 +143,7 @@ const BoardPage = (props: Props): JSX.Element => {
     }, [board?.title, activeView?.title])
 
     useEffect(() => {
-        let loadAction: any = initialLoad
+        let loadAction: any = initialLoad /* eslint-disable-line @typescript-eslint/no-explicit-any */
```

Comment:
I don't know if this would be needed, but maybe we can define `loadAction` as a function that matches both `initialLoad` and `initialReadOnlyLoad`. At the end of the day, it is a function that receive a `boardId` and returns an action to dispatch.